core/src/main/java/org/geysermc/geyser/registry/populator/BlockRegistryPopulator.java

```diff
@@ -284,7 +281,7 @@ private static void registerJavaBlocks() {
             String cleanJavaIdentifier = BlockUtils.getCleanIdentifier(entry.getKey());
             String bedrockIdentifier = entry.getValue().get("bedrock_identifier").asText();
 
-            if (!cleanIdentifiers.contains(cleanJavaIdentifier)) {
+            if (cleanIdentifiers.isEmpty() || !cleanIdentifiers.peekLast().equals(cleanJavaIdentifier)) {
```

Comment:
What's with this change?

Reply:
Looks like it can be simplified to `!cleanJavaIdentifier.equals(cleanIdentifiers.peekLast())`
The purpose of this it to avoid checking the entire list of cleanIdentifiers for every block state.

Reply:
Fair enough.